Makefile-bwrap.am

```diff
@@ -1,3 +1,15 @@
+noinst_PROGRAMS += generate-seccomp
+
+generate_seccomp_SOURCES=\
+	$(bwrap_srcpath)/generate-seccomp.c \
+	$(bwrap_srcpath)/utils.h \
+	$(bwrap_srcpath)/utils.c \
+	$(NULL)
+generate_seccomp_CFLAGS = $(AM_CFLAGS)
+generate_seccomp_LDADD = $(LIBSECCOMP_LIBS) $(SELINUX_LIBS)
+
+seccomp-filter.h: generate-seccomp$(EXEEXT)
+	./generate-seccomp$(EXEEXT) > seccomp-filter.h
```

Comment:
Does this work in a cross-compilation scenario?  Offhand it seems we'd generate rules for the wrong arch?